build/yamls/base/controller.yml

```diff
@@ -106,6 +108,42 @@ webhooks:
     timeoutSeconds: 5
 ---
 apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: "labelsmutator.antrea.tanzu.vmware.com"
+webhooks:
+  - name: "nsmutator.antrea.tanzu.vmware.com"
+    clientConfig:
+      service:
+        name: "antrea"
+        namespace: "kube-system"
+        path: "/mutate/nslabels"
+    rules:
+      - operations: ["CREATE", "UPDATE"]
+        apiGroups: [""]
+        apiVersions: ["v1"]
+        resources: ["namespaces"]
+        scope: "Cluster"
```

Comment:
since we have the same implementation (i.e. both Services and Namespaces are handled by `HandleMutationLabels`), can't we unify this a bit by using the same handler path (e.g. `/mutate/addlabels`) and having 2 entries in the `rules` array here (one for Namespaces and one for Services)? As opposed to defining 2 separate webhooks?